packages/cloud-cognitive/src/components/SidePanel/_side-panel.scss

```
@@ -264,9 +264,6 @@ $max-panel-size: 75%; // set max-width on max panels to 75%
       &.#{$action-set-block-class}--lg {
         @include setActionBarSize($large-panel-size);
       }
-      &.#{$block-class}__actions-container-condensed {
-        height: $layout-04;
-      }
     }
```

So, we can apply the required heights, for condensed or otherwise, to the buttons here. Something like this:

```suggestion
    }
    .#{$block-class}__actions-container .#{action-set-block-class}__action-button {
      height: $layout-05;
    }
    .#{$block-class}__actions-container.#{$block-class}__actions-container-condensed .#{action-set-block-class}__action-button {
      height: $layout-04;
    }    
```

---

PS -- is `height` the best style to be setting here? Button heights can vary based on their contents. The carbon Button sets `min-height` rather than `height`, and I think that might be better for us to do here too.

---

PPS -- this isn't the PR to open this issue on, but I notice we've got some SCSS nesting going on here, and I gather we should be following the carbon rule to avoid nesting of selectors. Just a heads up.

---

Ok, yes the nesting might need to be addressed in a separate PR. But yes, you're definitely right about that ^